eng/packages/General.props

```diff
@@ -18,6 +18,7 @@
     <PackageVersion Include="Microsoft.Extensions.VectorData.Abstractions" Version="$(MicrosoftExtensionsVectorDataAbstractionsVersion)" />
     <PackageVersion Include="Microsoft.IO.RecyclableMemoryStream" Version="3.0.0" />
     <PackageVersion Include="Microsoft.ML.Tokenizers" Version="$(MicrosoftMLTokenizersVersion)" />
+    <PackageVersion Include="ModelContextProtocol" Version="0.4.0-preview.2" />
```

Comment:
Just from glancing at the reader, I think this only needs ModelContextProtocol.Core?

Also, the latest version is preview.3, not preview.2.